integration-tests/azure-services-disabled/src/main/java/io/quarkiverse/azure/services/disabled/it/CosmosDisabledResource.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkiverse.azure.services.disabled.it;
+
+import static jakarta.ws.rs.core.Response.Status.NOT_FOUND;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosClient;
+
+@Path("/quarkus-azure-cosmos-disabled")
+@Produces(MediaType.TEXT_PLAIN)
+@ApplicationScoped
+public class CosmosDisabledResource {
+
+    @Inject
+    CosmosClient cosmosClient;
+
+    @Inject
+    CosmosAsyncClient cosmosAsyncClient;
+
+    @Path("/cosmosClient")
+    @GET
+    public Response getCosmosClient() {
+        assert cosmosClient == null : "The CosmosClient should be null";
```

Comment:
Does it mean `quarkus.azure.cosmos.enabled=false` here?

Reply:
Correct, see https://github.com/quarkiverse/quarkus-azure-services/pull/270/files#diff-edce8136f2b12f767ce5f710dddd111b6ea968ffe65c51080015601913e60e67R2.